### ISSUE TITLE
zoneState error experienced during installation of Home Assistant integration

### ISSUE THREAD

**Issue by mrlund** [jnsgruk/pytouchlinesl#6]
I'm installing the Home Assistant integration, and encountered the error: 
zones -> elements -> 9 -> zone -> zoneState unexpected value; permitted: 'zoneOff', 'noAlarm', 'zoneUnregistered' (type=value_error.const; given=noCommunication; permitted=('zoneOff', 'noAlarm', 'zoneUnregistered'))

It seems that it's receiving a state that it doesns't know, and that state is 'noCommunication'. I assume that it's caused by batteries that have run out in two of my (wireless) thermostats, but the error is preventing the whole integration from initializing, so none of my zones are available (and I'm out of batteries!). I would suggest that a nicer way to handle this would be to load all the zones regardless of this error on individual zones. 

I did take a look at the source and attempt a fix, but I don't work with python, and I can't run it locally which makes it hard to test :-) 
I don't know if this very simple fix is sufficient, or if there's other changes required downstream. I didn't think it was worthwhile submitting a PR over something so simple, especially since there may be other changes required. Anyway, here's the change: 

```
class ZoneAttributesModel(BaseModel):
   (other code omitted...)
    zone_state: Literal["zoneOff", "noAlarm", "zoneUnregistered", "noCommunication"] = Field(..., alias="zoneState")
   (other code omitted...)
```
So basically just adding the possible state value `noCommunication `in `pytouchlinesl\client\models\models.py:33`

**Comment by peroo** [comment]
I think your change is more or less all that is required. I've created a PR with some additional states, and updated the test fixtures to match.
